bin/common.sh

```diff
@@ -6,49 +6,87 @@ readonly clustersFile="$ENV_DIR/env/clusters.yaml"
 readonly helmfileOutputHide="(^\W+$|skipping|basePath=|Decrypting)"
 readonly helmfileOutputHideTpl="(^[\W^-]+$|skipping|basePath=|Decrypting)"
 readonly replacePathsPattern="s@../env@${ENV_DIR}@g"
+readonly otomiToolsImage="otomi/tools:1.4.10"
 
-get_k8s_version() {
+has_docker=$( docker --version &>/dev/null && echo "1" || echo "0")
+dind=0
```

Comment:
Can we not use 0/1 for booleans? We do have a style guide

Reply:
I copied the style from the existing codebase:
https://github.com/redkubes/otomi-core/blob/master/bin/otomi#L43-L45
Where can I find the style guide?

Reply:
In the redkubes/docs repo

Reply:
I read through the repo again, and only found: https://github.com/redkubes/docs/blob/master/development/principles.md Which does not mention boolean usage.

But I'll rewrite to true/false, if that has preference.